### ISSUE TITLE
Can't understand Chinese 

### ISSUE THREAD

**Issue by 00sapo** [MuiseDestiny/zotero-gpt#263]
### 这个问题是否已有issue

- [X] 我确认没有已有issue，且已阅读**常见问题**

### 运行环境

- **OS**: 
- **Zotero version**: 
- **Plugin version**: 

### 问题详情

Using Chinese prevents many people using this plugin to correctly report bugs and issues.

**Comment by zfb132** [comment]
The English template for reporting bugs and issues is supported. You can just use the following link.
https://github.com/MuiseDestiny/zotero-gpt/issues/new?assignees=&labels=&projects=&template=report_bug_en.yml

You can also click this button.
![image](https://github.com/MuiseDestiny/zotero-gpt/assets/18099238/50c08a31-ecbf-48cb-bc53-5ff5ba848902)

In addition to this, most components of zotero-gpt are in English.
* GUI in Zotero
* README of this plugin
* The output result of typing the `/help` command in the input box of the zotero-gpt window

**Comment by 00sapo** [comment]
I'm not a native English speaker, and I would be happy to use Italian in github. However, English is the most spoken language around the world (considering non-native speakers able to speak English). Until GitHub inserts an automatic translation feature, English remains mandatory for involving the largest community around the globe. Seeing code comments, commits, and most of the issues in Chinese discourages many people from contributing. I'm not against Chinese per se, but I think it's impractical. Unless your target are only Chinese people, which is ok for the readme (we could maintain a translated version in English in the wiki), but for anything else, it is better using English IMHO.

**Comment by zfb132** [comment]
Yes, you're right. We can gradually change the comments and commits to English. Also, we can maintain an English version for wiki and release notes. But it's a little difficult for users to use English to report bugs.

**Comment by li-il-li** [comment]
I agree with @00sapo . The plugin is not working for me right now, and I see (at least I think) people writing in chinese on issues about my error. BUT it is such a pita for me to debug my issue like this, that I gave up for now.
Love the idea and execution though! Will try it again at some point in time.
